client/core/core.go

```diff
@@ -4033,16 +4033,18 @@ func (c *Core) feeSuggestion(dc *dexConnection, assetID uint32) (feeSuggestion u
 	return dc.fetchFeeRate(assetID)
 }
 
-// Withdraw initiates a withdraw from an exchange wallet. The client password
-// must be provided as an additional verification.
-func (c *Core) Withdraw(pw []byte, assetID uint32, value uint64, address string) (asset.Coin, error) {
+// Send initiates either send or withdraw from an exchange wallet. if subtract
+// is true, fees are subtracted from the value else fees are taken from the
+// exchange wallet. The client password must be provided as an additional
+// verification.
+func (c *Core) Send(pw []byte, assetID uint32, value uint64, address string, subtract bool) (asset.Coin, error) {
```

Comment:
We need to first deprecate `Core`'s `Withdraw` before removing it.  There are Go consumers that use it, but I was less certain about the http api method.  It's trivial for `Withdraw` to call `Send` with `subtract=true`.  The comment for `Withdraw` should include `// Deprecated: ...` and we will note this in the release notes.

I don't mind killing off the method from the `client/asset.Wallet` interface, but we should be a bit more cautious with breaking changes to `Core`.

Reply:
Wouldn't any go code consuming `Core` fail to build if `Core` no longer has a `Withdraw` method? I suppose a rebuild would be necessary since the code will need to update the dcrdex module version after this is merged to access this change.

Reply:
Yes, that's what I'm getting at.  We shouldn't immediately remove a method without first deprecating it and marking it as such.